Scripts/DCS-BIOS/lib/modules/aircraft_modules/F4U-1D.lua

```diff
@@ -391,7 +391,11 @@ F4U_1D:definePotentiometer("RECEIVER_C26_SENSITIVITY_ROTARY", devices.RADIONAV,
 F4U_1D:defineFloat("RECEIVER_C26_FREQUENCY_DIAL", 94, { 0, 1 }, C26, "Receiver C-26 Frequency Dial")
 
 -- C-30A/ARC-5
--- local C30A = "C-30A/ARC-5 Control Unit"
+local C30A = "C-30A/ARC-5 Control Unit"
+
+F4U_1D:defineMultipositionSwitch("C30A_CHANNEL_SELECTOR", devices.RADIO, 3609, 88, 4, 1 / 3, C30A, "C30A Channel Seletor (1/2/3/4)")
+F4U_1D:defineToggleSwitch("C30A_ON_OFF_SWITCH", devices.RADIO, 3610, 92, C30A, "C30A On/Off Switch")
+F4U_1D:defineToggleSwitch("C30A_CW_VOICE_SWITCH", devices.RADIO, 3612, 95, C30A, "C30A CW/Voice Switch")
```

Comment:
To stay consistent there should be the corresponding in-cockpit values like the `C30A_CHANNEL_SELECTOR` description

```suggestion
F4U_1D:defineToggleSwitch("C30A_CW_VOICE_SWITCH", devices.RADIO, 3612, 95, C30A, "CW/Voice Switch (A/B/C/D)")
```

Reply:
@Blackhawkops could you take a look at this and see if this should be a 4-position switch like the channel selector?